render/query.go

```diff
@@ -344,7 +344,7 @@ func (r *Reply) getDataAggregated(ctx context.Context, cfg *config.Config, tf Ti
 	}
 
 	carbonlinkData := carbonlinkResponseRead()
-	data, err = parseAggregatedResponse(ctx, b, e, carbonlinkData, targets.isReverse)
+	data, err = parseAggregatedResponse(ctx, b, e, carbonlinkData, false)
```

Comment:
This should stay as it was, `targets.isReverse` is used to reverse metric names only from ClickHouse

Reply:
I's carbonlink response parse. Not Clickhouse.

Reply:
Here and in `getDataUnaggregated` `parse*ggregateResponse` reads everything. `carbonlinkData` is `extraPoints`, so parsed as is. `targets.isReverse` is used in parsing of ClickHouse response in `Scan()`